services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/evaluation/frustration/checker.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Evidence
+  module GenAI
+    module Evaluation
+      module Frustration
+        class Checker < Evaluation::BaseChecker
+          include Evidence::RegexListChecker
+
+          CATEGORY = 'frustration'
+          NOT_ALLOWED = {
+            frustrated: ['frustrated', 'frustration'],
+            addresses_feelings: ['you\'re feeling', 'I understand you', 'you are feeling']
+          }
+
+          def run = check_regex_list(feedback)
+
+          private def regex_config = NOT_ALLOWED
+          private def error_category = CATEGORY
```

Comment:
Maybe move  `private def error_category  = self.class::ERROR_CATEGORY` up to `BaseChecker`

Reply:
Good call.